packages/components/text-input/src/text-input.stories.ts

```diff
@@ -440,7 +439,7 @@ export const CustomValidation: StoryObj = {
           margin-bottom: 8px;
         }
       </style>
-      <sl-text-input required="true" .validators=${[validator]} error-size="sm"></sl-text-input>
+      <sl-text-input required .validators=${[validator]} error-size="sm"></sl-text-input>
```

Comment:
We don't want to always show that the input is valid (only on demand), eg. in this example we want to show only invalid state, but here when the input is valid, there is a background for `show-validity=valid` set for the wrapper, but it shouldn't be. :)

Reply:
Fixed!